ext_mongodb.php

```diff
@@ -665,6 +759,12 @@ function bsonUnserialize(array $data) : void;
 {
 }
 
+interface TypeWrapper {
```

Comment:
I think this is a duplicate interface definition (already declared at the top of the file).